### PR TITLE
Update dependencies; fix tests

### PR DESCRIPTION
**Pull request by zliang-akamai**
## 📝 Description

The dependencies are outdated possibly containing security vulnerabilities.  

## ✔️ How to Test

1. Create a linode.
2. `export TEST_TOKEN=YOUR_LINODE_TOKEN`
3. `export TEST_LABEL=the-lable-of-linode-you-created-in-step1`
4. `make test`

### CODE REVIEW

**Review by lgarber-akamai — APPROVED** [review]

**Comment by zliang-akamai** [comment]
@lgarber-akamai I bump up oauth2 and linodego after your review. Feel free to or not to re-review.

File: Makefile [hunk 1]
```diff
@@ -43,8 +43,8 @@ docker-login:
 
 build: $(PLUGIN_DIR)
 	# load plugin with versionied tag
-	docker plugin rm -f ${PLUGIN_NAME} 2>/dev/null || true
-	docker plugin create ${PLUGIN_NAME} ./$(PLUGIN_DIR)
+	# docker plugin rm -f ${PLUGIN_NAME} 2>/dev/null || true
+	# docker plugin create ${PLUGIN_NAME} ./$(PLUGIN_DIR)
```
Comment (yec-akamai) [inline Makefile:47]: Can we just remove these two lines?

**Review by yec-akamai — APPROVED** [review]
Looks good!

File: Makefile (hunk 1 above)
Comment (zliang-akamai) [reply Makefile:47]: I think we may want to keep it as a reference, just like `make deploy`.

**Pull request merged**